chainer/functions/normalization/_standardize.py

```diff
@@ -0,0 +1,100 @@
+import numpy
+import six
+
+from chainer import backend
+from chainer.backends import cuda
+from chainer import function_node
+import chainer.functions
+from chainer.utils import type_check
+
+
+class Standardize(function_node.FunctionNode):
+
+    """Standardization for `Weight standardization
+        <https://arxiv.org/abs/1903.10520>`_"""
+
+    def __init__(self, eps=1e-5):
+        self.eps = eps
+
+    def check_type_forward(self, in_types):
+        type_check.expect(in_types.size() == 1)
+        x_type, = in_types
+
+        type_check.expect(
+            x_type.dtype.kind == 'f',
+            x_type.ndim >= 2,
+        )
+
+    def _compute(self, xp, x):
+        # xp: numpy, cupy, or chainer.functions
+        axes = tuple(six.moves.range(1, len(x.shape)))
+        mu = xp.mean(x, axis=axes, keepdims=True)
+        x_mu = x - mu
+        squ_x_mu = xp.square(x_mu)
+        var = xp.mean(squ_x_mu, axis=axes, keepdims=True)
+        if xp is numpy:
+            std = numpy.sqrt(var, dtype=x.dtype) + self.eps
+        elif xp is cuda.cupy:
+            std = cuda.cupyx.sqrt(var, dtype=x.dtype) + self.eps
```

Comment:
`sqrt` is implemented in `cupy`, not `cupyx` (cupyx functions are not supported by numpy). I'm sorry to have you revise this section again and again.
```suggestion
            std = cuda.cupy.sqrt(var, dtype=x.dtype) + self.eps
```